### PR TITLE
fix: add eslint rules

### PR DESCRIPTION
**Pull request by 9snehal**

### CODE REVIEW

**Comment by 9snehal** [comment]
@abhishek71994  plz review my PR for eslint rules

File: view-src/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        ],
+        "semi": [
+            "error",
+            "always"
+        ],
+      "no-console": "off",
+      "no-undef": "off",
+      "no-unused-vars": "off",
+      "import/order": "off",
+      "camelcase": "off",
```
Comment (laveesh) [inline view-src/.eslintrc.js]: It is best practice to use camel case.
So set it on.

File: view-src/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      "eqeqeq": "off",
```
Comment (laveesh) [inline view-src/.eslintrc.js]: use of `eqeqeq` enforces for type completion.

File: view-src/.eslintrc.js (hunk 1 above)
Comment (9snehal) [reply view-src/.eslintrc.js]: ok @laveesh

**Comment by sweezy793** [comment]
@9snehal Please reference this PR with it's respective issue

**Comment by abhishek71994** [comment]
Fix for #48

**Pull request merged**